sdk/maps/azure-maps-service/samples/common/common.py

```diff
@@ -0,0 +1,50 @@
+from azure.maps.service import MapsClient
+import os
+import re
+import time
+
+
+def create_maps_client():
+    return MapsClient(os.environ["SUBSCRIPTION_KEY"], base_url='https://us.atlas.microsoft.com:443')
+
+
+def get_operation_location(poller, location_name):
+    return poller.polling_method(
+    )._initial_response.http_response.headers[location_name]
```

Comment:
This is using internal methods, which is not allowed. We make no guarantees that internal APIs stay compatible in new versions.

Reply:
Is there some way of getting the header from an operation in python?